pkg/skaffold/schema/v1beta9/upgrade_test.go

```diff
@@ -1,127 +0,0 @@
-/*
-Copyright 2019 The Skaffold Authors
-
-Licensed under the Apache License, Version 2.0 (the "License");
-you may not use this file except in compliance with the License.
-You may obtain a copy of the License at
-
-    http://www.apache.org/licenses/LICENSE-2.0
-
-Unless required by applicable law or agreed to in writing, software
-distributed under the License is distributed on an "AS IS" BASIS,
-WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-See the License for the specific language governing permissions and
-limitations under the License.
-*/
-
-package v1beta9
-
-import (
-	"testing"
-
-	"github.com/GoogleContainerTools/skaffold/pkg/skaffold/schema/latest"
-	"github.com/GoogleContainerTools/skaffold/testutil"
-	yaml "gopkg.in/yaml.v2"
-)
-
-func TestUpgrade(t *testing.T) {
-	yaml := `apiVersion: skaffold/v1beta9
-kind: Config
-build:
-  artifacts:
-  - image: gcr.io/k8s-skaffold/skaffold-example
-    plugin:
-      name: docker
-      properties:
-        dockerfile: path/to/Dockerfile
-  - image: gcr.io/k8s-skaffold/bazel
-    plugin:
-      name: bazel
-      properties:
-        target: //mytarget
-  executionEnvironment:
```

Comment:
This is gone in the v1beta8.upgrade_test. Is this intentional?

Reply:
Good catch - I think it should be in v1beta8.upgrade_test as we had plugins/executionEnv in v1beta8 and removing them in v1beta9
@priyawadhwa ?

Reply:
There should be tests for this now, thanks!